userspace/sysdig/chisels/statsd.lua

```diff
@@ -63,7 +63,7 @@ local function send(self, stat, delta, kind, sample_rate, neg)
     for s, v in pairs(stat) do
       if kind == 'c' then
         if type(s) == 'number' then
-          -- this is array or kyes ( increment{'register', 'register_accept'})
+          -- this is array or keys ( increment{'register', 'register_accept'})
```

Comment:
I believe this line should be "this is array of keys"

Reply:
```suggestion
          -- this is array of keys ( increment{'register', 'register_accept'})
```